.gitignore

```diff
@@ -28,3 +28,4 @@ tests/api-compatibility/reference/*.cs
 Novell
 *.patch
 *.keystore
+/omnisharp.json
```

Comment:
Does this line need a leading `/`?

Reply:
We create the file in the root directory only, I figured other directories may get their own locally created config files and we shouldn't hide them